### PR TITLE
feat: dump-metadata upload to dump-server including auth

### LINKED ISSUE #3: save the created dump's metadata on dump-server
invoke post request to the [dump-server](https://github.com/MapColonies/dump-server) for saving the newly created dump's metadata

### PR DESCRIPTION
**Pull request by melancholiai**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

Closes #3

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ name = "pypi"
 osmeterium = "==0.0.3"
 boto3 = "==1.17.20"
 MapColoniesJSONLogger = "==1.0.0"
+requests = "*"
```
Comment (syncush) [inline Pipfile]: stick to a specific requests version

File: README.md [hunk 2]
```diff
@@ -45,6 +54,8 @@ Required if `UPLOAD_TO_OBJECT_STORAGE` is true:
     - s3_bucket_not_exist: 105 - the given bucket name does not exist on the object storage.
     - object_key_already_exists: 106 - the created dump has an object key which does already exist on the bucket.
     - missing_env_arg: 107 - missing a required environment argument.
+    - dump_server_upload_error: 108 - an error occurred in the process of uploading metadata to the dump-server.
+    - dump_server_request_error: 109 - an error occurred in the process of sending the request to the dump-server.
```
Comment (syncush) [inline README.md]: lets change to markdown table

**Review by syncush — DISMISSED** [review]

File: Pipfile (hunk 1 above)
Comment (melancholiai) [reply Pipfile]: done.

File: README.md (hunk 2 above)
Comment (melancholiai) [reply README.md]: done.

**Review by syncush — APPROVED** [review]

**Pull request merged**